config/jobs/kubernetes/sig-testing/bazel-build-test.yaml

```diff
@@ -312,6 +312,11 @@ postsubmits:
       preset-service-account: "true"
       preset-bazel-scratch-dir: "true"
       preset-bazel-remote-cache-enabled: "true"
+    skip_branches:
+    - release-1.14 # per-release job
+    - release-1.13 # per-release job
+    - release-1.12 # per-release job
+    - release-1.11 # per-release job
```

Comment:
something like release-\d+.\d+? :smirk:

Reply:
This follow the pattern of the rest of the file... 

Let me send a follow up that converts the files to do it this way.